examples/jacobi/jacobi_component/row_range.hpp

```diff
@@ -1,10 +1,11 @@
-
 //  Copyright (c) 2012 Thomas Heller
 //
 //  SPDX-License-Identifier: BSL-1.0
 //  Distributed under the Boost Software License, Version 1.0. (See accompanying
 //  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
+// hpxinspect:noinclude:hpx::intrusive_ptr
```

Comment:
I think you could change the inspect to check to expect `hpx/modules/memory.hpp` instead. Same for `row.hpp`.

Reply:
Ok I wasn't sure if we wanted to change all those includes in the main hpx files, will do :)

Reply:
Don't forget to remove the `hpxinspect` comments now that they're not needed anymore.